source/hic_hal/stm32/stm32h743ii/sdk.c

```diff
@@ -209,3 +220,169 @@ static uint32_t tim2_clk_div(uint32_t apb1clkdiv)
     }
 }
 
+/* MSP stands for MCU support package and comes from STCube. The MSP layer is responsible
+ * for initializing low level hardware like GPIOs and clocks for some peripherals.
+ * This function is called by the HAL, but we could also make our drivers handle this functionality
+ * with more refactoring.
+ */
+
+void HAL_MspInit(void)
+{
+    __HAL_RCC_SYSCFG_CLK_ENABLE();
+}
+
+/**
+* @brief PCD MSP Initialization
+* This function configures the hardware resources used in this example
+* @param hpcd: PCD handle pointer
+* @retval None
+*/
+void HAL_PCD_MspInit(PCD_HandleTypeDef* hpcd)
```

Comment:
I was looking for where this gets called, but didn't spot it.  
IIUC, this is a Pheripheral Controller Driver ?  for USB?  
https://www.st.com/resource/en/user_manual/dm00105879-description-of-stm32f4-hal-and-ll-drivers-stmicroelectronics.pdf